2022-04-21 15:10:34,016 INFO    MainThread:303205 [wandb_setup.py:_flush():71] setting env: {}
2022-04-21 15:10:34,016 INFO    MainThread:303205 [wandb_setup.py:_flush():71] setting login settings: {}
2022-04-21 15:10:34,016 INFO    MainThread:303205 [wandb_setup.py:_flush():71] setting login settings: {}
2022-04-21 15:10:34,016 INFO    MainThread:303205 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220421_151034-3vj2br9d/logs/debug.log
2022-04-21 15:10:34,016 INFO    MainThread:303205 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220421_151034-3vj2br9d/logs/debug-internal.log
2022-04-21 15:10:34,017 INFO    MainThread:303205 [wandb_init.py:init():404] calling init triggers
2022-04-21 15:10:34,017 INFO    MainThread:303205 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-04-21 15:10:34,017 INFO    MainThread:303205 [wandb_init.py:init():460] starting backend
2022-04-21 15:10:34,017 INFO    MainThread:303205 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-21 15:10:34,032 INFO    MainThread:303205 [backend.py:ensure_launched():216] starting backend process...
2022-04-21 15:10:34,036 INFO    MainThread:303205 [backend.py:ensure_launched():221] started backend process with pid: 303260
2022-04-21 15:10:34,037 INFO    MainThread:303205 [wandb_init.py:init():469] backend started and connected
2022-04-21 15:10:34,039 INFO    MainThread:303205 [wandb_init.py:init():533] updated telemetry
2022-04-21 15:10:34,046 INFO    MainThread:303205 [wandb_init.py:init():563] communicating current version
2022-04-21 15:10:34,382 INFO    MainThread:303205 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.14 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-21 15:10:34,382 INFO    MainThread:303205 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-04-21 15:10:34,563 INFO    MainThread:303205 [wandb_init.py:init():606] starting run threads in backend
2022-04-21 15:10:34,633 INFO    MainThread:303205 [wandb_run.py:_console_start():1810] atexit reg
2022-04-21 15:10:34,634 INFO    MainThread:303205 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-04-21 15:10:34,634 INFO    MainThread:303205 [wandb_run.py:_redirect():1689] Redirecting console.
2022-04-21 15:10:34,635 INFO    MainThread:303205 [wandb_run.py:_redirect():1745] Redirects installed.
2022-04-21 15:10:34,635 INFO    MainThread:303205 [wandb_init.py:init():633] run started, returning control to user process
2022-04-21 15:10:37,617 INFO    MainThread:303205 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-04-21 15:10:37,618 INFO    MainThread:303205 [wandb_run.py:_restore():1752] restore
2022-04-21 15:10:39,685 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 950
  total_bytes: 950
}

2022-04-21 15:10:40,014 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 950
  total_bytes: 950
}

2022-04-21 15:10:40,590 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 950
  total_bytes: 2631
}

2022-04-21 15:10:40,693 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 950
  total_bytes: 8108
}

2022-04-21 15:10:40,795 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8108
  total_bytes: 8108
}

2022-04-21 15:10:40,898 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8108
  total_bytes: 8108
}

2022-04-21 15:10:41,001 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8108
  total_bytes: 8108
}

2022-04-21 15:10:41,105 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8108
  total_bytes: 8108
}

2022-04-21 15:10:41,208 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8108
  total_bytes: 8108
}

2022-04-21 15:10:41,313 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8108
  total_bytes: 8108
}

2022-04-21 15:10:41,509 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8108
  total_bytes: 8108
}

2022-04-21 15:10:41,915 INFO    MainThread:303205 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8108
  total_bytes: 8108
}
local_info {
}

2022-04-21 15:10:42,999 INFO    MainThread:303205 [wandb_run.py:_append_files():2180] logging synced files
